src/components/TopNav/PrimaryNav.js

```diff
@@ -36,7 +37,7 @@ const PrimaryNav = ({
         <a
           className={cn(styles.tcLogo, collapsed && styles.tcLogoPush)}
           onClick={onClickLogo}
-          href='/'
+          href={loggedIn ? '/my-dashboard' : '/'}
```

Comment:
Please don't hardcoded the URL, put this inside config file.

Use URL.BASE to non logged user and add a new config to logged user.